sql/internal/sqlx/exclude.go

```diff
@@ -140,6 +140,15 @@ func excludeT(t *schema.Table, pattern string) (err error) {
 		}
 		return filepath.Match(pattern, fk.Symbol)
 	})
+
+	t.Attrs, err = filter(t.Attrs, func(c schema.Attr) (bool, error) {
+		t := c.(*schema.Check)
+		match, err := filepath.Match(pattern, t.Name)
+		if !match || err != nil {
+			return false, err
+		}
+		return true, nil
+	})
```

Comment:
Skip the attribute if it's not a `Check` type.
```suggestion
	t.Attrs, err = filter(t.Attrs, func(c schema.Attr) (bool, error) {
		c, ok := c.(*schema.Check)
		if !ok {
			return false, nil
		}
		match, err := filepath.Match(pattern, t.Name)
		if !match || err != nil {
			return false, err
		}
		return true, nil
	})
```